uber/templates/preregistration/paid_preregistrations.html

```diff
@@ -29,18 +29,25 @@
 
 
 <div class="row">
-  <div class="well col-md-offset-1 col-md-10">
-    <h1>Registration Complete</h1>
-    <p>
-      Congratulations! Your registration was successful.
-    </p>
-    <p>
-      Your credit card was successfully charged for
-      <strong>${{ total_cost }}</strong>, and you will receive a confirmation
-      email sent to you shortly. If you paid for other registrations, those
-      attendees will get their own confirmation email.
-    </p>
-  </div>
+    <div class="well col-md-offset-1 col-md-10">
```

Comment:
This is actually definitely related. When a promo code made an attendee's badge free the message used to be "Your credit card was successfully charged for $0". Which is strange and not actually true. Here we check if the total cost is zero, and display a different message if it is.